dao-protocol/contracts/facets/ekoDAO/AdminFacet.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "./LibDiamond.sol";
+
+bytes32 constant DIAMOND_STORAGE_POSITION =
+        keccak256("diamond.standard.admin.storage");
+
+struct AppStorage {
+    address tokenAddress;
+
+}
+
+struct AdminState {
+        address[] admins;
+        address[] superAdmins;
+        address[] acceptedStableCoins;
+    }
+
+   
+contract AdminFacet {
+      AppStorage internal s;
+
+    // Mapping to store list of administrators
+    mapping(address => bool) private administrators;
+    // Mapping to store list of super administrators
+    mapping(address => bool) private superAdmins;
+    // Mapping to store list of accepted ekoStable tokens
+    mapping(string => bool) private acceptedEkoStables;
```

Comment:
Accepted stable tokens should be a mapping of the address of stable token,  on adding a stable token the contract should deploy a new stable token (Ekostable) with the features of that token, i.e digits, name, and symbol (should have the Eko prefix eg EkoUSDC and Ekolance USD Coin) etc.